include/deal.II/lac/sparse_direct.h

```diff
@@ -480,7 +481,6 @@ class SparseDirectMUMPS : public EnableObserverPointer
        * algorithm, an algorithm with higher compression than the standard one.
        */
       bool blr_ucfs;
-
       /**
```

Comment:
I think these empty lines were right. Would you mind adding them back in here and below again?

Reply:
Now we have empty lines between member variables of `AdditionalData` (as suggested by this commit). I don't see what are the other empty lines I should address.